src/components/Firealerts/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Suspense } from "react";
+import { useFirealertsTriggers } from "./api/internal/useFirealertsTriggers";
+import { EmulatorDisabled } from "../common/EmulatorDisabled";
+import { Spinner } from "../common/Spinner";
+import { useIsEmulatorDisabled } from "../common/EmulatorConfigProvider";
+import { FirealertsProvider } from "./api/useFirealerts";
+import { Switch } from "react-router";
+import { Route } from "react-router-dom";
+import { FirealertsForm } from "./Form/FirealertsForm";
+
+export const FirealertsRoute: React.FC = () => {
+  return (<Suspense fallback={<FirealertsRouteSuspended />}>
+    <HydrateFirealerts>
+      <Switch>
+        <Route path="/firealerts">
```

Comment:
Is this needed?

Reply:
No, in a previous iteration the UI was going to use different routes, but that ended up getting removed and it looks like some code is left over.